QUEST.tsv

```diff
@@ -3147,7 +3147,7 @@ QUEST_20151224_003154	(The pillar shows an image of a goddess with the number 3.
 QUEST_20151224_003155	(The pillar shows an image of a human with the number 3.)
 QUEST_20151224_003156	(The pillar shows an image of a Demon Goddess with the number 3.)
 QUEST_20151224_003157	...Do you see it?{nl}
-QUEST_20151224_003158	This is the catastrophe that already fell upon us.{nl}But an ever greater calamity is just beginning.{nl}
+QUEST_20151224_003158	This is the catastrophe that already fell upon us...{nl}But an ever greater calamity is just the beginning.{nl}
```

Comment:
But an ever greater calamity is just the beginning.

But an **even** greater calamity is **just beginning**.